Extension/src/LanguageServer/configurations.ts

```diff
@@ -403,6 +403,8 @@ export class CppProperties {
         }
 
         result["workspaceFolderBasename"] = this.rootUri ? path.basename(this.rootUri.fsPath) : "";
+        result["execPath"] = process.execPath;
```

Comment:
Sorry for iterating one comment at a time, but there are a couple more places that need to be updated:
https://github.com/microsoft/vscode-cpptools/blob/5dcb3081e502f84185b71f773f3d106b5d27f63c/Extension/src/LanguageServer/client.ts#L759-L761
https://github.com/microsoft/vscode-cpptools/blob/5dcb3081e502f84185b71f773f3d106b5d27f63c/Extension/src/LanguageServer/configurations.ts#L1205-L1208
https://github.com/microsoft/vscode-cpptools/blob/5dcb3081e502f84185b71f773f3d106b5d27f63c/Extension/c_cpp_properties.schema.json#L193

Reply:
thank you for finding those.